schedule/jeos/sle/minimalvm-main.yaml

```diff
@@ -25,6 +25,8 @@ conditional_schedule:
                 - installation/bootloader_uefi
             'aarch64':
                 - installation/bootloader_uefi
+            'ppc64le-p10':
+                - installation/bootloader_uefi
```

Comment:
I assume the failure happens as there are no serial consoles enabled. In that case we need to run `jeos/host_config` or `console/system_prepare` before this module

Reply:
`openqa-clone-custom-git-refspec https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/22729 https://openqa.suse.de/tests/18476207 SCHEDULE=tests/installation/bootloader_uefi,tests/jeos/host_config,tests/microos/verify_setup TEST+=-jeos-host_config`
-> https://openqa.suse.de/tests/18514834
`openqa-clone-custom-git-refspec https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/22729 https://openqa.suse.de/tests/18476207 SCHEDULE=tests/installation/bootloader_uefi,tests/console/system_prepare,tests/microos/verify_setup TEST+=-console-system_prepare`
-> https://openqa.suse.de/tests/18514835

Reply:
ofc when the needle should assert tty6 and the test never moved from tty1
https://openqa.suse.de/tests/18514835#step/system_prepare/2